http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -3,6 +3,53 @@ set -e
 
 # If you update this file, please update azure.custom-script-extension.file-uris in reference.conf so that Leonardo can adopt the new script
 
+# Formatting and mounting persistent disk
+WORK_DIRECTORY='/home/jupyter/persistent_disk'
+echo $WORK_DIRECTORY
+
+## The PD should be the only `sd` disk that is not mounted yet
+AllsdDisks=($(lsblk --nodeps --noheadings --output NAME --paths | grep -i "sd"))
+echo ${AllsdDisks[@]}
+FreesdDisks=()
+echo ${FreesdDisks[@]}
+for Disk in "${AllsdDisks[@]}"; do
+    echo $Disk
+    Mounts="$(lsblk --noheadings --output MOUNTPOINT "${Disk}" | grep -vE "^$")"
+    echo $Mounts
+    echo "${Mounts[0]}"
```

Comment:
I might need the help of a bash or azure vm custom script wizzard. This line (and the rest up to mounting that I cannot perform due to permission issues) works perfectly fine from a VM jupyter terminal but somehow fails in this script

Reply:
It may be helpful for debugging to make a new machine with port 22 open and run these commands, not sure if you've directly connected to a machine yet.
If you need some help with getting this set up LMK